doc/user/content/integrations/http-api.md

```diff
@@ -52,6 +52,11 @@ The HTTP API provides two modes with slightly different transactional semantics
     - Eagerly commits DDL (e.g. `CREATE TABLE`) in implicit transactions, but
       not DML (e.g. `INSERT`).
 
+### OpenAPI spec
+
+Download our [OpenAPI](https://swagger.io/specification/) v3 spec for this
+interface: [environmentd-openapi.yml](static/environmentd-openapi.yml).
```

Comment:
This link to the yml file doesn't seem to work, and I don't see any examples of including an image in a markdown file, maybe someone can point me in the right direction?